### ISSUE TITLE
Opencart Account Banned

### ISSUE THREAD

**Issue by gshockfactory** [opencart/opencart#2950]
Hi guys, i really do not know where else to resort so i hope posting here would help me get this problem solved. I tried using Opencart helpdesk but all im getting is a reply from Daniel Kerr saying "YOURE BANNED! YOURE BLACKLISTED! YOURE PLACED IN OUR FRAUD CATEGORY"

Well it all started about 2 weeks ago when my NEWLY created PayPal account was limited as most new PayPal accounts will somehow down the road get limited after receiving large amount of payments and they would ask for verifications such as Photo ID, Utility Bill, Suppliers Contact Info & Proof of Delivery. That is what happened to my account and at the same time, they also randomly pulled back 5 transactions for review. 3 of that transactions were opencart and what i realized was they just pulled the newest 5 transactions and so happened i bought a few modules of Opencart. Please bare in mind i have been buying modules of Opencart with the same account for more than 2 years now and this was the first time this happened.

Daniel Kerr assumed i opened the dispute myself but i have been trying so hard to tell him IT WAS NOT ME who opened the dispute. There are 4 kinds of funds hold on PayPal as below:
1) Item not as described (Opened by the buyer)
2) Item not received (Opened by the buyer)
3) Credit Card Chargeback (Done by the buyer through his/her credit card company)
4) Review by PayPal (Done by PayPal and NOT by the buyer. Buyer has no control over this)

On April 21st they decided to reverse all 5 transactions and all the money went back to my account. 1 day after pulling back the transactions, they decided to restore my account. I have the email to proof that my account is fine now and able to receive/send funds to Daniel Kerr. I also attached this screenshot on the Opencart support forum. For the other 2 transactions which were people who sold services to me, they easily agreed that i repay them and we solved all problems. However Daniel Kerr of Opencart does not seem to want this. I have told him numerous times to please send me an invoice to the same PayPal account and i will send him the funds right away. I am not looking to pay for modules and then do a chargeback after installing it on my website. I do not believe in cheating people these way. If the module works, i'll pay for it.

I have also used the Opencart refund system once which was successful and he should know i am fully aware of the Opencart refund system which is flawless. So why would i open a dispute when i'm fully aware on how to get a refund via the Opencart support system. Doesn't make sense.

I have had this happened to me as a seller numerous times as well and still happens until today. I receive a payment then few days after i reiceve an email with the title "Please respond within 7 days" and in that email it states that PayPal believes this transaction may not be authorized bla bla bla so on. However i know the buyer is not in fault because PayPal has some strict standards so its a normal thing mainly for new accounts and for accounts that have not passed the first step verification process.

I really hope Daniel Kerr would read through this thoroughly before deciding to reply me with one liners. Please be professional and hear me out. I have been trying to tell you IT WAS NOT ME WHO OPENED THE DISPUTE and I AM WILLING TO PAY YOU BACK FOR ALL 3 TRANSACTIONS since the money is back in my account sitting there and i have not used it and reserved it to pay you back. I am also asking you to send the invoice to the same PayPal account and not some other different account. This clearly shows i am not a fraud and it was just a misunderstanding. There is really no need to suspend my account when i have almost 20 modules i paid for in there over the past years. This is really an unreasonable  way to handle this. I really hope you will change the way you handle this stuff and learn to differentiate between a dispute opened by the buyer and a freeze placed by PayPal to review the funds.

Here is the screenshot of the disputes. You can clearly see it says Inquiry by PayPal.  This means PayPal did the dispute and not me.
![screenshot_1](https://cloud.githubusercontent.com/assets/12098610/7318455/b3b7e10a-eabe-11e4-9dcf-af58c7132f53.png)

Here is the email where the day my PayPal account was limited which was April 7 (Surprisingly they said my account had customers reporting that they did not receive orders however that account is so new that not even 1 dispute has been opened on it.)
![screenshot_6](https://cloud.githubusercontent.com/assets/12098610/7318349/9fa556bc-eabd-11e4-845c-cf9700c407f3.jpg)

Here is another email after that where they decided to link me to another PayPal account that i have NO IDEA ABOUT and say that i owe them $500. After a few phone calls to them, they realized they were in the wrong and i have no connections to  this account whatsoever and that is why they decided to restore my account again.
![screenshot_7](https://cloud.githubusercontent.com/assets/12098610/7318364/d4bd8626-eabd-11e4-81bb-6587c863486f.jpg)

Here is the screenshot of the email PayPal sent me on April 22 where they decided to restore my account access back and your transactions was pulled back on April 21st. I do not know why PayPal decided to do it this way but i guess PayPal can really be dicks at times as well and there is why www.paypalsucks.com exists but we somehow can't live without them because they are the most convenient way to pay and get paid.
![screenshot_5](https://cloud.githubusercontent.com/assets/12098610/7318325/3ab71c68-eabd-11e4-8835-07a672b45ab0.jpg)

You guys should know how strict PayPal is and for them to restore my account clearly shows them i am NOT A FRAUD. I wish Daniel Kerr could see this. I am not trying to cheat you, why would i want to when your marketplace has helped my business so much :(

Thank you.

**Comment by fabdo** [comment]
In the screenshots above, i am the person that had the $30 transaction reversed. I am an Opencart module developer, and have worked with this client for many months now. Every payment has been smooth, we have had over 10 transactions and only 1 transaction that is the $30 as shown above got held lately however he has paid me again and have kept me updated throughout the whole situation. I can vouch that this guy is not a scammer.

**Comment by danielkerr** [comment]
I don't think your telling the truth here. i will look into this and will reply.

if you think the issue tracker is the right place to post stuff you are wrong!

in fact this will get you banned!

**Comment by gshockfactory** [comment]
What do you feel i am not telling the truth about? Its 100% the truth man. I have showed you screenshots. Tell me what you need me to show you and i will show it to you, i have nothing to hide.

I'm sorry as i have mentioned above i have few hundred worth of extensions on that account and this is a nightmare if i lose it all. Please understand where i am coming from.

**Comment by danielkerr** [comment]
support ticket answered by my staff:

Matt Wednesday 15:32
Hi,

it seems like Paypal has pulled back all the 3 transactions, and its back into my account. However my PayPal account is still limited. How can we proceed now? If you would like me to repay all these payments, please sent me an invoice to my other paypal account nette.teh29@gmail.com or you can remove those modules from my download list and i will repurchase them again.

Thank you.


Matt Monday 13:01
I seem to be having a never ending nightmare with them. They have linked my account to some other paypal account which I have no idea about and are forcing me to pay around 800usd which they say I owe them.

I've been telling them that I have no knowledge of that account or whatsoever but they are not being helpful and sending me automated replies only.

That is why I'm contacting you to say most probably the payments will be reversed to my account back but what I can do is pay the amount owed with my family members paypal account or you can just remove the extensions that the payments was reversed back.

Once again please know it was not me who did the charge backs. I fully am aware of opencarts refund policy and have used it before so it makes no sense that I would do a chargeback. I really hope I can get my account unlocked for the meantime as I have many extensions I have bought over the years and would like to purchase a few more currently.

I googled about this problem and see that many people hve gotten this happening to them all of a sudden.


Felix Monday 12:55
Hi Matt,

I check there are 3 Reversed order from your opencart account. 
So, our system banned your account.

Please try get your paypal account solve with paypal and let us know. 
We can help you to unbanned the account when you cancel the display from paypal.

Best Regards 
Felix 
OpenCart Team.


Matt Monday 01:06
Would appreciate your quick response on this matter as i am waiting to buy a few more modules but my account is currently blocked on Opencart due to this :(

Please understand the disputes was not opened by me. If it was, i could easily close it right now and solve all of these however PayPal limited my account and pulled back 5 random transactions and so happened it was for Opencart as well.


Matt Apr 14 02:31
Contact Details

Name: Matt 
E-Mail: gshockfactory@gmail.com 
IP: 60.52.7.219

Enquiry:

Hi,

It seems like PayPal is not going to solve my account problem any time soon. Is there anyway for me to pay for the transactions that they pulled back with another of my PayPal account so i can get access back to my opencart account?

Thank you.

**Comment by danielkerr** [comment]
this is paypal reversing the transactions even after i sent them your communication

Dear Kerr Daniel,

We've concluded our investigation into the transaction detailed below.

We've returned the funds to the PayPal account holder, and your account
will be debited for the amount of this transaction.

-----------------------------------
Helping ensure safe transactions
-----------------------------------

Most of the payments you receive will be from honest buyers. PayPal wants
to help ensure that your transactions are as safe as possible.

You can often reduce your risk by paying close attention to payment details
and unusual requests.

* Watch for address inconsistencies:
It's fairly common for a buyer's postal address to be different from the
billing address. In some cases, though, it could indicate fraud. For
instance, a high-priced item that has a billing address in one country and
a postal address in another may be suspicious.

* Beware of unusual requests that could indicate suspicious activity, such
as:
- Buyers who want rush delivery at any cost
- Buyers who send partial payments from different PayPal accounts
- Do not make payment in full

Go to our Security page for more information.

-----------------------------------
Transaction Details
-----------------------------------


Buyer's name: Platinum GS Trading Enterprise
Buyer's email: platinumgstrading@gmail.com
Transaction ID: 91T06869XT774521S

Transaction date: 4 Apr 2015
Transaction amount: $29.99 USD
Invoice ID: 598418 - Matt Hehe
Your transaction ID: 9YB090568S0473450
Case number: PP-003-832-086-152


Thanks,
PayPal
Please do not reply to this email. This mailbox is not monitored and you
will not receive a response. For assistance, log in to your PayPal account
and click the Help link in the top right corner of any PayPal page.

**Comment by gshockfactory** [comment]
Yes i did read every reply your staff replied me. However after this reply is when PayPal decided to put the money back into my account. If you want i can show you the screenshot of my main account, the limitation has been removed. PayPal works in weird ways man trust me, i am not lying at all here. Everything i've told you is the truth. The only module i am having an open issue now with is the Nitro Pack. If i were to do a dispute, don't you think i would do a dispute for the Nitro Pack? The 3 extensions that PayPal pulled back i have no problems with it at all. I am using them on my website with no problem and am happy with all this modules.

That is why i feel the need to send you the money i owe you for those modules as i am using them currently.

**Comment by gshockfactory** [comment]
The email that you got above, i get it too from buyers who pay me, but that doesn't mean the buyer did a chargeback. Its just that PayPal thinks that payment is high risk due to the account being a new account or due to that account being limited after a short while that payment was sent. They usually pull back a few transactions so that if anything goes really wrong, they will have funds in that account to issue a refund to customers who have sent me money. However as you can see i posted the email paypal sent on April 22 that my account has been restored, i have solved every issue with that account. So happens that they decided to reverse the funds sent to Opencart on the 21st of April.

I did call them and tell them to please release the funds that are being held but they told me "We are sorry but that funds can only be released once you solve your limitation so i had no choice but to wait not knowing they were going to reverse it"

**Comment by danielkerr** [comment]
exactly how paypal works since i have over 400 transaction a day with them.

your account would have been unlocked if you had contacted me and sent me the money. 

but posting here and lying that I said:

"Daniel Kerr saying "YOURE BANNED! YOURE BLACKLISTED! YOURE PLACED IN OUR FRAUD CATEGORY""

I have never said this to you!

you want to lie about me!! 

now your account is permanently banned!

**Comment by gshockfactory** [comment]
Someone has been using your name and replying to my suports then. Please check my support under my account gshockfactory(at)gmail.com

Here is a screenshot.

I have been sending support emails endlessly telling them please send me an invoice as i would like to pay them the money i owe. I have been doing this since a week back.
![screenshot_8](https://cloud.githubusercontent.com/assets/12098610/7321013/740abd20-ead2-11e4-80d1-70e56565474a.jpg)

**Comment by danielkerr** [comment]
there is only one ticket for gshockfactory@gmail.com and i have never replied to it.

nor can i find other tickets with the multiple emails you have used to buy from me.

**Comment by gshockfactory** [comment]
You are big corporation, your PayPal accounts are basically white listed by PayPal however we small companies that are new to PayPal go through hell with them. This is why i understand you might not know how PayPal treat new accounts hence why the long post above and why i have been trying to explain my whole situation to you through Support and never gotten a reply. All i got was an automated reply so i resorted to posting it here. 

I am sorry if this was the wrong place to post. Please do accept my apologies.

There must be something wrong because i change my email from naomikx at hotmail.com to gshockfactory at hotmail.com a few weeks back. So some supports has been going to the first email and some going to the second.

**Comment by haxcop** [comment]
Sorry guys but i think this is a place to report BUGS and FIXES to the code of Opencart FOR DEVELOPERS and not mean to be used as a complain desk. 

So please be more accurate for the adequate channels.

Yours faithfully 
Carlos Lozada

**Comment by stockmaster** [comment]
What is funny and ironic is, Daniel will close all relevant Issues and PRs here without 1 WORD of reply. And yet he goes in lenght when things are out of scope.

**Comment by danielkerr** [comment]
I don;t have to explain the reasons why i close issues!

many reasons why they get closed with out a response. normally because the issues are time wasters.

**Comment by gshockfactory** [comment]
I have replied to your support ticket, please do send me the methods on how to pay you pack for the amount i owe you. Also if a MOD could remove this topic i would be grateful.

Once again sorry everyone for posting this irrelevant topic here. My sincere apologies.

**Comment by stockmaster** [comment]
Like the issues i posted, and i tested in LENGHT to be sure i wouldnt be wasting anyones time.
Regarding the lack of email notification to the recipiet of the Vourcher cards?
I even tested it in the demo store, to be SURE and SURE, but nop, not even a 
"i cant replicate this in any way".
I do agree, some things are stupid. but i am unsure which is more stupid:
Wasting time finding and testing bugs, of wasting time Reporting them.

**Comment by danielkerr** [comment]
@stockmaster, yes and the issue was with the mail system, that has been reported or already fixed.  not and issue with the vouchers.

**Comment by stockmaster** [comment]
@danielkerr I tell what. Because we are all busy people. 
LOOK for more than 10 secs at https://github.com/opencart/opencart/issues/2937
Tested on 2.0.2.1Rc, where the mail system IS fixed already. LATEST patched build used to test it.
Prove me i wasted your time and mine, and you wont see a single further comment from my part regarding opencart code.
I know you get 100s of issues, but please dont say we waste our time when we spend hours trying to test things to report. WE all want the same here. WE want a) to help you b) we want a solid work platform.
Please assume that principle and not that we all take our time to waste yours
Regards

**Comment by ManaKill2** [comment]
Amen @stockmaster

**Comment by gshockfactory** [comment]
@danielkerr Could you please edit or remove your posts on this issue as it contains several of my personal information. TQ.

**Comment by gshockfactory** [comment]
Suck a dick whiteboi

On 24/4/2015 10:24 PM, Daniel Kerr wrote:
>
> I don't think your telling the truth here. i will look into this and 
> will reply.
>
> if you think the issue tracker is the right place to post stuff you 
> are wrong!
>
> in fact this will get you banned!
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/opencart/opencart/issues/2950#issuecomment-95947366>.
>



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus
